HybrasylXML/XSD/Castables.Designer.cs

```diff
@@ -33,7 +33,7 @@ public partial class Castable
 
         private Categories _categories;
 
-        private CastCostList _castCosts;
```

Comment:
This should follow the pattern of List<CastCost> as is part for all other lists. Please do not create new types for items that should be generic.

Reply:
and, shit, logged in with my work account, again.

Reply:
No clue what you mean. If you read the diff, the difference is we are making it into a `List<CastCost>` when previously, it was not.

Reply:
I read this backwards. Color reversal is a thing, I swear.